codebuild/spec/buildspec_fuzz_scheduled.yml

```diff
@@ -0,0 +1,88 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+batch:
+  build-matrix:
+    static:
+      env:
+        privileged-mode: true
+    dynamic:
+      env:
+        compute-type:
+          - BUILD_GENERAL1_LARGE
+        image:
+          - 024603541914.dkr.ecr.us-west-2.amazonaws.com/docker:ubuntu22codebuild
+        privileged-mode: true
+        variables:
+          S2N_LIBCRYPTO: 
+            - awslc
```

Comment:
Was this always scheduled to only run on awslc, or did we change that recently? I thought this job predated awslc, or at least predated awslc being our default libcrypto

Reply:
We changed this recently. iirc the old job was running with openssl-1.0.2-FIPS or openssl-1.1.1. I updated this to be tested with awslc to be able to fuzz test with our default libcrypto. We could add more but since scheduled fuzz runs a batch job for ~7 hours, adding additional libcrypto here will be an expensive decision.

Reply:
Was the old job running with both openssl-1.0.2 and openssl-1.1.1, or do you not know which one it was running with? Can you look up old runs of the job to check their batchspec for what libcrypto they ran with? Like, what was the batchspec 3 months ago?

If we were only running on one libcrypto before too, then yes awslc is probably a better choice than either of the others. But if we were running on more than one before, we can probably afford to at least add back openssl-1.1.1.

Reply:
Unfortunately, we can’t view the build history since I removed the old job when transitioning to a batch job. The previous setup had ~30 build triggers for individual tests, which implies we were only running with one libcrypto, given that we have about 30 fuzz tests total. Based on that, I’ll stick with awslc for this job.